.github/workflows/deploy_conda.yaml

```diff
@@ -0,0 +1,56 @@
+name: Conda Deployment
```

Comment:
I think it would be a good idea to merge this with the deploy workflow because they are essentially triggered under the same circumstances.

Reply:
Yes, I think GHA can still run them as separate "jobs", right?

Reply:
sure. You can even make the jobs depend on each other. That might allow us to only run the tests once.